### PR TITLE
Show images full screen in overlay on click

### PR DESCRIPTION
**Pull request by trevorgerhardt**

### CODE REVIEW

**Comment by abyrd** [comment]
First impression is that it looks good, background blur could be a little less strong, and it would be nice if the enlarged image closed when clicking outside the image, not just inside it. Clicking outside to close seems like a common interaction on other sites.

**Comment by trevorgerhardt** [comment]
> First impression is that it looks good, background blur could be a little less strong, and it would be nice if the enlarged image closed when clicking outside the image, not just inside it. Clicking outside to close seems like a common interaction on other sites.

Done 👍

File: custom.css [hunk 1]
```diff
@@ -25,17 +25,36 @@
...
-.GrowImage:active {
+.ImageOverlayBackdrop {
   cursor: zoom-out;
-  max-height: unset;
+  position: fixed;
+  top: 0;
+  left: 0;
+  height: 100vh;
+  width: 100vw;
+  backdrop-filter: blur(5px);
```
Comment (abyrd) [inline custom.css:44]: Personally I like the background dimmed and less blurred, like this.
```suggestion
  backdrop-filter: brightness(60%) blur(2px);
```

**Review by abyrd — APPROVED** [review]
Looks good. Personally I like the background dimmed and less blurred (I suggested a change inline). I find the dimming makes the image stand out more from the background, without a jolting change in the sharpness of the background. But this is all pretty subjective.

**Pull request merged**